presto-main/src/main/java/io/prestosql/sql/planner/LocalExecutionPlanner.java

```diff
@@ -521,6 +525,9 @@ private static void addLookupOuterDrivers(LocalExecutionPlanContext context)
         private final List<DriverFactory> driverFactories;
         private final Optional<IndexSourceContext> indexSourceContext;
 
+        private TupleDomain<DynamicFilters.Descriptor> dynamicFilter = TupleDomain.all();
```

Comment:
nit: add a comment that this potentially contains domains for dynamic filters for different table scans (in case of colocated joins)

Reply:
Done (at `LocalDynamicFiltersCollector`).